contents/docs/integrate/_snippets/install-api.mdx

```diff
@@ -0,0 +1,37 @@
+Personal API keys enable full access to your account, just like e-mail address and password. You can create any number of them and each one can be invalidated individually. This makes for greater control for you and improved security of stored data.
+
+#### How to obtain a personal API key
+
+1. Go to [Account settings](https://app.posthog.com/me/settings)
+2. In the 'Personal API Keys' section, click "+ Create a Personal API Key".
+3. Give your key a label – this is just for you, usually to describe the key's purpose.
+4. At the top of the list you should see your brand new key. **Immediately** copy its value, as you'll **never** see it again after refreshing the page.
+
+You can create as many keys as you like.
+
+![How to create an api key](../../../images/docs/api/create-api-key.mp4)
+
+#### How to use the personal API key
+
+There are three options:
+
+1. Use the `Authorization` header and `Bearer` authentication, like so:
+    ```js
+    const headers = {
+        Authorization: `Bearer ${POSTHOG_PERSONAL_API_KEY}`
+    }
+    ```
+2. Put the key in request body, like so:
+    ```js
+    const body = {
+        personal_api_key: POSTHOG_PERSONAL_API_KEY
+    }
+    ```
+3. Put the key in query string, like so:
+    ```js
+    const url = `https://app.posthog.com/api/event/?personal_api_key=${POSTHOG_PERSONAL_API_KEY}`
+    ```
+
+Any one of these methods works, but only the value encountered first (in the order above) will be used for authentication.
+
+For PostHog Cloud US, use `app.posthog.com` as the host. For PostHog Cloud EU, use `eu.posthog.com`.
```

Comment:
```suggestion
```
I don't think this makes sense to have here. Is probably better explained somewhere else.